src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(new ColorSupplier().getRandomColor());
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. Here, a new instance of ColorSupplier is created every time the getRandomBall() method is called. Consider creating a single instance of ColorSupplier as a class-level field and reusing it.